src/Blosc.jl

```diff
@@ -175,12 +175,13 @@ decompress(::Type{T}, src::DenseVector{UInt8}) where {T} =
 # If `nthreads` is 1, the the serial version is chosen and a possible previous existing pool is ended.
 # If this function is not callled, `nthreads` is set to 1 internally.
 """
-    set_num_threads(n=Sys.CPU_CORES)
+    set_num_threads(n=Sys.Threads.nthreads())
```

Comment:
Really, this should have nothing to do with `Sys.Threads.nthreads()`, since Blosc uses its own threads, unrelated to the Julia threads.

One of these days I (or someone) needs to get around to rewriting this package to use Blosc2, which supports a plug-in mechanism that will allow it to use Julia's own threads and task schedule, making it composable with other Julia threading.

Reply:
It sounds like we should then change this to `n=1` since the Julia and Blosc's threading are unrelated?

Reply:
Looks like there's already progress wrapping Blosc2 (https://github.com/waralex/Blosc2.jl)